pkg/apiserver/apiserver.go

```diff
@@ -137,23 +137,33 @@ func (g *APIGroupVersion) InstallREST(container *restful.Container) error {
 // this method will return an error.
 func (g *APIGroupVersion) UpdateREST(container *restful.Container) error {
 	installer := g.newInstaller()
-	var ws *restful.WebService = nil
-
-	for i, s := range container.RegisteredWebServices() {
-		if s.RootPath() == installer.prefix {
-			ws = container.RegisteredWebServices()[i]
-			break
-		}
-	}
-
+	ws := g.GetREST(container)
 	if ws == nil {
 		return apierrors.NewInternalError(fmt.Errorf("unable to find an existing webservice for prefix %s", installer.prefix))
 	}
 	apiResources, registrationErrors := installer.Install(ws)
+	// TODO: This panics when the route is the last one in the webservice.
```

Comment:
This = ?
Also can you explain what the TODO is? What should this be replaced with?